.github/templates/docker_builds_ci_workflow.yml.j2

```diff
@@ -48,10 +48,6 @@ jobs:
           retry docker pull "${DOCKER_IMAGE}"
       !{{ common.parse_ref() }}
       !{{ common.teardown_ec2_linux() }}
-      - name: Hold runner for 2 hours or until ssh sessions have drained
```

Comment:
These were leftover and the functionality is now included in `teardown_ec2_linux`